src/components/Trip/index.js

```diff
@@ -0,0 +1,3 @@
+import TripView from './TripView'
+
+export default TripView
```

Comment:
nit: I really don't understand what is the goal and the gain of that kind of file. This is not the first I saw it. Can you explain to me?

Reply:
C'est pour pouvoir faire des imports sans répéter le composant qui a le même nom que le dossier. Pour éviter de faire `import TripView from 'src/components/TripView/TripView'` et faire directement `import TripView from 'src/components/TripView'`